robosuite/devices/spacemouse.py

```diff
@@ -101,37 +102,26 @@ def convert(b1, b2):
     return scale_to_control(to_int16(b1, b2))
 
 
-class SpaceMouse(Device):
+class SpaceMouseSdkWrapper:
```

Comment:
maybe call this SpaceMouseListener instead

Reply:
Renamed.